python/ray/serve/api.py

```diff
@@ -232,6 +238,23 @@ def create_backend(
 
         if config is None:
             config = {}
+        if ray_actor_options is None:
+            ray_actor_options = {}
+        if env is None:
+            # If conda is activated, default to conda env of the driver.
+            if os.environ.get("CONDA_PREFIX"):
+                ray_actor_options.update(override_environment_variables={
+                    "PYTHONHOME": os.environ.get("CONDA_PREFIX")
+                })
+        else:
+            if not os.environ.get("CONDA_PREFIX"):
+                raise ValueError("conda must be activated to use CondaEnv.")
+            python_home = os.path.join(
+                os.path.split(os.environ.get("CONDA_PREFIX"))[0], env.name)
+            if not os.path.isdir(python_home):
+                raise ValueError("conda env " + env.name + " not found.")
```

Comment:
I think we should split this out into a utility function and add some basic tests for it.